contrib/dokku-installer.rb

```diff
@@ -3,21 +3,26 @@
 require "open3"
 
 if ARGV[0] == "onboot"
-  File.open("/etc/init/dokku-installer.conf", "w") do |f|
-    f.puts "start on runlevel [2345]"
-    f.puts "exec #{File.absolute_path(__FILE__)} selfdestruct"
-  end if File.directory?("/etc/init")
-  File.open("/etc/systemd/system/dokku-installer.service", "w") do |f|
-    f.puts "[Unit]"
-    f.puts "Description=Dokku web-installer"
-    f.puts ""
-    f.puts "[Service]"
-    f.puts "ExecStart=#{File.absolute_path(__FILE__)} selfdestruct"
-    f.puts ""
-    f.puts "[Install]"
-    f.puts "WantedBy=multi-user.target"
-    f.puts "WantedBy=graphical.target"
-  end if File.directory?("/etc/systemd/system")
+  if File.realpath("/sbin/init") != "/sbin/init" then
```

Comment:
Ok, I can explain - On ubuntu it works same, but on older debian or older ubuntu releases it switching from upstart/systemd to sysv-init. This code: File.realpath("/sbin/init") != "/sbin/init" detects sysvinit or upstart/systemd by checking if /sbin/init is symlink to systemd/upstart or it is regular file. Is it more clear? I dont push on you, I am ok with PR refuse. This PR solves problem on Debian Wheezy, where its impossible to install dokku without manual modifications.